packages/react-admin-date-picker/src/DatePicker.tsx

```diff
@@ -33,7 +33,7 @@ export const DatePicker: React.FC<IProps> = ({
     }, [localeName]);
 
     return (
-        <sc.SingleDatePickerWrapper fullWidth={fullWidth} color={color}>
+        <sc.SingleDatePickerWrapper fullWidth={fullWidth} color={color} onBlur={(event: React.FocusEvent<HTMLInputElement>) => onBlur(event)}>
```

Comment:
```suggestion
        <sc.SingleDatePickerWrapper fullWidth={fullWidth} color={color} onBlur={onBlur}>
```
.. wäre nicht so verbose

Reply:
passt hab ich umgeändert